Index: setup.py
===================================================================
--- setup.py	(revision 6146)
+++ setup.py	(working copy)
@@ -1,17 +1,16 @@
 #!/usr/bin/env python
 import sys, os
-from distutils.core import setup
+from setuptools import setup
 
 setup(name="odtwriter",
-    version="1.3d",
-    description="convert rst to ODF/odt/odp.",
-    author="Dave Kuhlman",
-    author_email="dkuhlman@rexx.com",
-    packages=['docutils.writers.odtwriter'],
-    package_dir={'docutils.writers.odtwriter': 'odtwriter'},
-    package_data={'docutils.writers.odtwriter': ['styles.odt', ]},
-    scripts=["tools/rst2odt.py", "tools/rst2odt_prepstyles.py"],
-    )
+      version="1.3d_eggified",
+      description="convert rst to ODF/odt/odp.",
+      author="Dave Kuhlman",
+      author_email="dkuhlman@rexx.com",
+      packages=['odtwriter.writers.odtwriter'],
+      package_dir={'odtwriter.writers.odtwriter': 'odtwriter'},
+      package_data={'odtwriter.writers.odtwriter': ['styles.odt', ]},
+      scripts=["tools/rst2odt.py", "tools/rst2odt_prepstyles.py"],
+      install_requires=['docutils'],
+      license='BSD')
 
-
-
